clar2wasm/tests/wasm-generation/principal.rs

```diff
@@ -0,0 +1,192 @@
+//
+// Proptests that should only be executed
+// when running Clarity::V2 or Clarity::v3.
+//
+#[cfg(not(feature = "test-clarity-v1"))]
+mod clarity_v2_v3 {
+    use clar2wasm::tools::crosscheck;
+    use clarity::util::hash::hex_bytes;
+    use clarity::vm::types::{
+        BuffData, OptionalData, PrincipalData, QualifiedContractIdentifier, SequenceData,
+        StandardPrincipalData, TupleData,
+    };
+    use clarity::vm::Value;
+    use proptest::prelude::{Just, Strategy};
+    use proptest::{prop_oneof, proptest};
+
+    use crate::{buffer, runtime_config, PropValue};
+
+    fn create_principal_from_strings(
+        version_string: &str,
+        principal_string: &str,
+        name: Option<&str>,
```

Comment:
nit: `contract_name`? just to make it clear it refers to the contract

Reply:
Changed.